src/energyplus/Test/ScheduleInterval_GTest.cpp

```diff
@@ -1569,7 +1569,7 @@ TEST_F(EnergyPlusFixture, DISABLED_ForwardTranslator_ScheduleVariableInterval_Da
   bool until24Found = false;
   bool nextValueShouldBeLast = false;
   bool hitNextValueShouldBeLast = false;
-  unsigned numUntils = 0;
+  [[maybe_unused]] unsigned numUntils = 0;
```

Comment:
fix some `-Wunused-but-set-variable` on apple clang 13.1.6 (earlier versions won't have the issue)